.github/workflows/node-ci-push.yml

```diff
@@ -0,0 +1,120 @@
+name: Node CI (landed on master)
```

Comment:
This was copied over from https://github.com/Khan/wonder-stuff/blob/main/.github/workflows/node-ci-main.yml as @somewhatabstract suggested!.

The only addition is the `Chromatic` workflow.